modules/credentials/providers.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = "0.13.7"
+  required_version = ">= 0.13.7, < 0.15.0"
```

Comment:
TIL

Reply:
We learned that too.

Reply:
We probably need to backport this to AWS.